python/cudf/cudf/pandas/fast_slow_proxy.py

```diff
@@ -1011,7 +1018,33 @@ def _transform_arg(
         # use __reduce_ex__ instead...
         if type(arg) is tuple:
             # Must come first to avoid infinite recursion
-            return tuple(_transform_arg(a, attribute_name, seen) for a in arg)
+            if (
+                len(arg) > 0
+                and isinstance(arg[0], _MethodProxy)
+                and arg[0].__customqualname__ in _SPECIAL_FUNCTIONS_ARGS_MAP
+            ):
+                indices_map = _SPECIAL_FUNCTIONS_ARGS_MAP[
+                    arg[0].__customqualname__
+                ]
+                method_proxy, cust_args, cust_kwargs = arg
+
+                cust_args = tuple(
+                    _transform_arg(a, "_fsproxy_slow", seen)
+                    if i - 1 in indices_map
+                    else _transform_arg(a, attribute_name, seen)
+                    for i, a in enumerate(cust_args)
+                )
+                cust_kwargs = _transform_arg(cust_kwargs, attribute_name, seen)
+                res = (
```

Comment:
Return this directly.
```suggestion
                return (
```